learningequality/studio
# Pagination hotfix

## Summary
### Description of the change(s) you made
This PR fixes #3005 which is an issue with channels briefly disappearing.
It also includes Sairina's updates to the Studio issue template.


### Manual verification steps performed

For 3005: 
Initial test
1. Added a new channel
2. Navigate back to channel page
3. All channels continued to appear, with none disappearing

### Does this introduce any tech-debt items?
Yes, this would revert pagination. As @micahscopes noted, this merits more discussion about whether or not there are backend considerations here, or if it is mainly a frontend UX/navigability issue.

___
## Reviewer guidance
### How can a reviewer test these changes?

1. Add a new channel
2. Navigate back to channel page
3. All channels should continue to appear (none should disappear from the bottom)

### Are there any risky areas that deserve extra testing?

@metodimilevqa - there has been some discussion on the dev side if reverting this pagination feature will cause performance issues. We will discuss more on Monday, so some additional testing with an eye on performance would be extremely informative. Would you be able to test this several times using accounts that have varying numbers of channels? For example, perhaps one account <25 channels, one with between 25 - 50 channels, and one with > 50 channels? Thank you for your help on this!


### Contributor's Checklist


PR process:


Studio-specifc:



Testing:

___

### Reviewer's Checklist
#### This section is for reviewers to fill out.
